src/us/kbase/workspace/test/kbase/SampleIDHandlerFactoryTest.java

```diff
@@ -331,7 +331,9 @@ public void sampleIDHandlerProcessIDs() throws Exception {
 		final SampleServiceClientWrapper cli = mock(SampleServiceClientWrapper.class);
 		
 		final IdReferenceHandler<Integer> h = new SampleIdHandlerFactory(cli)
-				.createHandler(Integer.class, new AuthToken("t", "user1"));
+				// keep the new String(). String interning caused this test to pass when
+				// it should've failed
+				.createHandler(Integer.class, new AuthToken("t", new String("user1")));
```

Comment:
Pretty subtle bug

Reply:
it's subtle here, but in the integration tests which are coming in the next PR, it caused authorization for the sample owner to fail